features/base/conference/components/Conference.js

```diff
@@ -1,8 +1,8 @@
 import React, { Component } from 'react';
 
-import { FilmStrip } from '../../filmStrip';
-import { LargeVideo } from '../../largeVideo';
-import { Toolbar } from '../../toolbar';
+import { FilmStrip } from '../../../filmStrip';
+import { LargeVideo } from '../../../largeVideo';
+import { Toolbar } from '../../../toolbar';
```

Comment:
@tsareg, we do not import non-base features into base features.

I think that could be 1 signal that renaming features/conference to features/base/conference doesn't implement a working abstraction. And I believe that's because you've tried to combine 2 different abstractions that just happen to be named with 1 name:

1. The old features/conference represents the abstraction of a page which displays a VoIP conference. This feature is unlikely to ever need to be a base feature because no other feature is likely to want to import it but the app itself which wants to navigate from the welcome page to the conference page.

2. The new features/base/conference adds the abstraction similar/corresponding to lib-jitis-meet's JitsiConference. Given that our app is built on top of lib-jitsi-meet and multiple JitsiConference-related events are very likely to affect different portions of the application's state and UI, you've correctly identified it as a feature that very likely needs to be based.

I think we may choose to (1) live with features/base/conference and features/conference i.e. with 1 name or (2) come up with 2 separate names. I don't see why we cannot do the change from 1 name to 2 names later on so I don't have (2) high on my priority list.